rest/taskana-rest-spring/src/main/java/pro/taskana/rest/ClassificationController.java

```diff
@@ -1,31 +1,23 @@
 package pro.taskana.rest;
```

Comment:
Do not use initial commit as commit comment, type something which describes the commit